dockerfiles/init/manifests/che.env

```diff
@@ -431,59 +431,66 @@ CHE_SINGLE_PORT=false
+#        PVC with name that is evaluated as '{che.infra.kubernetes.pvc.name} + '-' + {generated_8_chars}'
 #        will be used for storing of workspaces data.
 #        Existing PVC will be used or a new one will be created if it doesn't exist.
-#CHE_INFRA_OPENSHIFT_PVC_STRATEGY=common
+#CHE_INFRA_KUBERNETES_PVC_STRATEGY=common
 
 # Defines whether to run a job that creates workspace's subpath directories in persistent volume for the 'common' strategy before launching a workspace.
 # Necessary in some versions of OpenShift/Kubernetes as workspace subpath volume mounts are created with root permissions,
 # and thus cannot be modified by workspaces running as a user (presents an error importing projects into a workspace in Che).
-# The default is "true", but should be set to false if the version of Openshift/Kubernetes creates subdirectories with user permissions.
+# The default is "true", but should be set to false if the version of Kubernetes/Kubernetes creates subdirectories with user permissions.
 # Relevant issue: https://github.com/kubernetes/kubernetes/issues/41638
 # Note that this property has effect only if the 'common' PVC strategy used.
-#CHE_INFRA_OPENSHIFT_PVC_PRECREATE__SUBPATHS=true
+#CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS=true
 
 # Defines the name of Persistent Volume Claim for che workspace.
-#CHE_INFRA_OPENSHIFT_PVC_NAME=claim-che-workspace
+#CHE_INFRA_KUBERNETES_PVC_NAME=claim-che-workspace
 
 # Defines the size of Persistent Volume Claim of che workspace.
 # Format described here:
-# https://docs.openshift.com/container-platform/latest/dev_guide/compute_resources.html#dev-compute-resources
-#CHE_INFRA_OPENSHIFT_PVC_QUANTITY=10Gi
+# https://docs.kubernetes.com/container-platform/latest/dev_guide/compute_resources.html#dev-compute-resources
+#CHE_INFRA_KUBERNETES_PVC_QUANTITY=10Gi
 
 # Defines Persistent Volume Claim access mode.
 # Note that for common PVC strategy changing of access mode affects the number of simultaneously running workspaces.
 # If OpenShift flavor where che running is using PVs with RWX access mode then a limit of running workspaces at the same time
 # bounded only by che limits configuration like(RAM, CPU etc).
 # Detailed information about access mode is described here:
-# https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/storage.html#pv-access-modes
-#CHE_INFRA_OPENSHIFT_PVC_ACCESS__MODE=ReadWriteOnce
+# https://docs.kubernetes.com/container-platform/latest/architecture/additional_concepts/storage.html#pv-access-modes
+#CHE_INFRA_KUBERNETES_PVC_ACCESS__MODE=ReadWriteOnce
+
+########################################################################################
+#####                                                                              #####
+#####                               Openshift Infrastructure                       #####
+#####                                                                              #####
+#
+
```

Comment:
nothign for openshift?

Reply:
OpenShift will reuse k8s infra configuration properties. I'll doc it. 
OpenShift section will have only OpenShift specific properties like `che.infra.openshift.project`